### ISSUE TITLE
max amps on four conners for powering the ESCs

### ISSUE THREAD

**Issue by Racing4funn** [SpektreWorks/spektreworks.github.io#8]
Cant find these numbers..

Thanks

**Comment by SpektreWorks** [comment]
The maximum recommended current draw, per the documentation, is 140A with 280A surges. See the [Power](http://docs.spektreworks.com/carrier_board_v1_3_1/#power) section of the documentation.

**Comment by Racing4funn** [comment]
Where is the RC in to connect my TBS Crossfire?

 

 




 

 

-----Original Message-----

**Comment by SpektreWorks** [comment]
Assuming you're using PPM or SBUS, the connectors are shared on the AUX rail. See the image here: http://docs.spektreworks.com/carrier_board_v1_3_1/#aux-pins

**Comment by Racing4funn** [comment]
do I have too mount your board on anti-vibration legs?

 

 




 

 

-----Original Message-----

**Comment by SpektreWorks** [comment]
No, the Pixhawk Cube and carrier board should be hard-mounted directly to the frame. All of the vibration sensitive sensors are internally isolated in the cube.

**Comment by Racing4funn** [comment]
Any advice where I can find this cable?  Thanks!

**Comment by jaspholm** [comment]
Have you tested the impacts of sustained 140 A current draw on the board, say for 50 minutes? What sort of maximum surface temperatures on the board you recommend to be on the safe side. I know 280 A surges are ok but I'm talking about prolonged operations close to 140 A and frequently exceeding this. 

I would also be interested to see a section added in the manual for pre-flight/post-flight checks, i.e. is there anything specific that you recommend for checking (either visually or by using some software) to ensure the board is functioning correctly and that one captures potential wear and tear on the board prior failures. 

Lastly, have you done studies on how noisy the board is and if there is any need to further shield the cube from emf the board causes?
